problems/1.types+operators.js

```diff
@@ -2,16 +2,23 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+function isNumber(val) {
+  if (typeof(val) === "number"){
+    return true
+  }else{
+    return false
+  }
 }
 
 /** Takes in a value and returns whether it is truthy
 * @param {any} val - The input value
 * @returns {boolean} - Is val truthy
 */
-function isTruthy() {
-
+function isTruthy(val) {
+  while (val === !true || val !== true){
```

Comment:
this doesn't check if val is truthy, it just checks if it is equal to true which is a different problem.

Look into the bang operator (!) and how you can use it to solve this problem